src/system.cr

```diff
@@ -22,4 +22,14 @@ module System
   def self.cpu_count
     Crystal::System.cpu_count
   end
+
+  # Returns the short user name of the currently logged in user or `nil` if it
+  # can't be determined. Note that this information is not secure.
```

Comment:
Re: nil, there's no guarantee that `getlogin` returns a string, it can return NULL. That's the source of the comment - so I'm not certain how to handle that particualr behavior.